doc/usage.rst

```diff
@@ -98,6 +98,12 @@ few command line options:
    to parse ``setup.py``. This flag can be provided multiple times to
    pass multiple packages for pre-install.
 
+.. cmdoption:: --pip-tool <exename>
+
+   Executable that will be used to install requirements after the
+   preinstall stage.  Usually you'll install this program by using the
+   ``--preinstall`` argument. By default vanilla pip is used.
```

Comment:
...commenting to myself here: Even that wording can leave a small possibility to confusion: what if user supplies `--upgrade-pip` or `--preinstall=pip==x.y.z`, then it no longer is the submitted pip :)

Maybe we just remove the last sentence? It's sort of implied anyway.